src/components/MessageList/MessageList.tsx

```diff
@@ -783,10 +781,9 @@ const MessageListWithContext = <
         flatListRef.current.scrollToIndex({
           index: indexOfParentInMessageList - 1,
         });
+        setTargetedMessage(messageId);
       }
-
-      setTargetedMessage(messageId);
-    } else {
+    } catch (e) {
       loadChannelAtMessage({ messageId });
```

Comment:
Our previous logic was flawed, if message is not within viewable frame, then it would reload the list. Which means if you press quoted message, the parent message NEEDS to be in current window frame ... which means at distance of approx max 5-10 messages. So instead we should just let react-native take care of this logic for us. If it fails, we loadChannel at parent message.